### ISSUE TITLE
False-positive E:-002

### ISSUE THREAD

**Issue by GoogleCodeExporter** [google/closure-linter#65]
```
What steps will reproduce the problem?

Lint this file:
define(
    function() {
        return {
            doSomething: function() {
                var x = '';
                return x;
            }
        };
    }
);

What is the expected output? What do you see instead?

There should not be an error, but I get this one:
Line 8, E:-002: Error parsing file at token "<JavaScriptToken: 8, }, "}", {}, 
MetaData(None)>". Unable to check the rest of file.
Error "None"

What version of the product are you using? On what operating system?

2.3.10, Mac OSX Mountain Lion
```

Original issue reported on code.google.com by `visusnor...@gmail.com` on 10 May 2013 at 8:44
